main.go

```diff
@@ -171,17 +198,20 @@ func fetchMetricFamilies(url string, ch chan<- *dto.MetricFamily) {
 		}
 	}
 }
-
 func main() {
-	runtime.GOMAXPROCS(2)
-	if len(os.Args) != 2 {
-		log.Fatalf("Usage: %s METRICS_URL", os.Args[0])
-	}
-
+	runtime.GOMAXPROCS(4)
+	url := flag.String("url", "https://example.com", "url")
```

Comment:
I think you want to keep this as it was.  We don't want to change the behavior of the existing script.